sql/core/src/main/scala/org/apache/spark/sql/DataFrameStatFunctions.scala

```diff
@@ -72,13 +73,38 @@ final class DataFrameStatFunctions private[sql](df: DataFrame) {
   }
 
   /**
+   * Calculates the approximate quantiles of numerical columns of a DataFrame.
```

Comment:
I'd prefer to add a `@see` reference to the above method doc. This should work: `@see [[DataFrameStatsFunctions.approxQuantile(col:Str* approxQuantile]] for detailed description.` - I remember I checked that the API doc worked with this ([here](https://github.com/apache/spark/pull/12207/files#r59176931)) but can you double check?